File: challenges/2-tabuada/js/naygo/tabuada.js
// Input/Args: 2
// Output: tabuada até o número inserido

const prompt = require('prompt-sync')();

function Tabuada(numero) {

    console.log('===========================')

    for (let i = 1; i <= numero; i++) {
        console.log(`A tabuada do número ${i} é:`)

        for (let j = 1; j <= 10; j++) {
            console.log(i + '*' + j + ' = ' + (i * j));
        }
        console.log('===========================')
    }

}

let number = prompt('Digite um numero: ');
Tabuada(number)

File: challenges/2-tabuada/python/luizeduardobp/tabuada.py
numero=int(input("Digite um numero: "))
for i in range(1, numero+1):
    print("=====================================")
    print("A tabuada do numero {} é:".format(i))
    for e in range(1, 10+1):
        print("[{}] X [{}] = {}".format(i,e,i*e))
